### ISSUE TITLE
Where is GT labels in test set?

### ISSUE THREAD

**Issue by Seonghoon-Yu** [OpenCausaLab/MORE#4]
Hello, 

I wish to measure the model's accuracy on the test set of the More dataset. 

However, the test set only provides questions and options, without the ground truth labels, making it impossible to measure accuracy.

Where can I obtain the ground truth labels for the test set?

**Comment by chenmeiqii** [comment]
Hi! Thank you for your interest in the MORE dataset. To ensure a fair and consistent evaluation protocol, we do not publicly release the ground-truth labels for the test set. For your model development and evaluation, we recommend using the validation set, which has the same distribution as the test set and is fully labeled.

**Comment by Seonghoon-Yu** [comment]
Thanks for replying.

Can i get your accuracy on validation set?
In the paper, the test accuracy is available whereas the validation accuracy is not present.
I need to compare ours with them.

Thanks.

**Comment by chenmeiqii** [comment]
> Thanks for replying.
> 
> Can i get your accuracy on validation set? In the paper, the test accuracy is available whereas the validation accuracy is not present. I need to compare ours with them.
> 
> Thanks.

Sorry that we did not retain the validation-set results and therefore cannot provide the exact validation accuracy.

At this point, there are two options:

1. You may evaluate on the validation set directly for comparison.

2. We are currently discussing how to release the test set or its evaluation protocol, which may take about 1–2 days. We will update you once ready.

**Comment by chenmeiqii** [comment]
Hi, the test set has been uploaded and is available for download at https://huggingface.co/datasets/OpenCausaLab/MORE/viewer/default/test.

**Comment by Seonghoon-Yu** [comment]
Thanks for releasing the test set
